core/right.service.js

```diff
@@ -0,0 +1,18 @@
+//const Promise = require('promise');
+//const winston = require('winston');
+//const logger = winston.loggers.get('gomngr');
+const CONFIG = require('config')
+
+// This will allow to change the way admin right are checked without impact on other part of the code
+// It will also be used to manage new role in future feature
+
+exports.is_admin = is_admin;
+
+async function is_admin(uid) {
```

Comment:
Why async? Not needed here..   for future async ops?

Reply:
Yes I think about put the admin list in mongo database or maybe in an ldap group (maybe as an option or several option)